core/models/user.py

```diff
@@ -1,4 +1,5 @@
 import uuid
+import hashlib
```

Comment:
function md5 is referenced below, but md5 is a deprecated library.  hashlib is the current library and has an md5 routine, so I choose to update to this library.